cilium-operator.Dockerfile

```diff
@@ -7,8 +7,12 @@ ARG V
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o cilium-operator
 RUN strip cilium-operator
 
+FROM docker.io/library/alpine:3.9.3 as certs
+RUN apk --update add ca-certificates
+
 FROM scratch
 LABEL maintainer="maintainer@cilium.io"
 COPY --from=builder /go/src/github.com/cilium/cilium/operator/cilium-operator /usr/bin/cilium-operator
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
Is it enough just to copy `/etc/ssl/certs/ca-certificates.crt`? Asking, as the package contains more certs: https://pkgs.alpinelinux.org/contents?file=&path=&name=ca-certificates&branch=v3.9&repo=main&arch=x86.

Reply:
Yes, that file is supposed to contain all certificates.